BlueDBOnDisk/src/main/java/org/bluedb/disk/BlueDbOnDisk.java

```diff
@@ -107,7 +113,7 @@ protected BlueCollectionOnDisk getUntypedCollectionForBackup(String folderName)
 			collection = collections.get(folderName);
 		}
 		if (collection == null) {
-			collection = new BlueCollectionOnDisk(this, folderName, null, Serializable.class);
+			collection = new BlueCollectionOnDisk(this, folderName, null, Serializable.class, Arrays.asList());
```

Comment:
You meant to pass in an empty list here right? I think you did, just want to make sure

Reply:
yeah